CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 ## 2.0.9
-Improved error message when missing authentication parameters.
+* Improved error message when missing authentication parameters.
+* fix base_url slash inconsistency
```

Comment:
It's configuration.host and not base_url

Reply:
sure, fixing it.

Reply:
I tested generic_request_func directly with the next options:
1. base/ + /path
2.base + / path
3. base + path
4.base/ + path